src/libraries/dirs.proj

```diff
@@ -1,26 +1,37 @@
 <Project>
-  <Import Project="$(MSBuildThisFileDirectory)Directory.Build.props" />
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
 
-  <!-- required to build the projects in their specified order -->
   <PropertyGroup>
-    <SerializeProjects>true</SerializeProjects>
+    <TargetFramework>$(BuildTargetFramework)</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup Condition="'$(DirectoryToBuild)' == ''">
-    <Project Include="ref.builds" />
-    <Project Include="shims\generated\*.csproj" />
-    <Project Include="Native\native-binplace.proj" />
-    <Project Include="src.builds" />
-    <Project Include="shims\manual\*.csproj" />
-    <Project Include="shims\ApiCompat.proj" Condition="'$(DotNetBuildFromSource)' != 'true' and '$(BuildTargetFramework)' == '$(NetCoreAppCurrent)'" />
-    <Project Include="pretest.proj" Condition="'$(DotNetBuildFromSource)' != 'true' and '$(BuildAllConfigurations)' != 'true'" />
+    <ProjectReference Include="ref.proj" />
+    <ProjectReference Include="shims\generated\*.csproj" />
```

Comment:
Can you move this into ref.proj and build as another target after the main build?  That would make ref.proj more useful, even though it does make it slightly more complicated.

Reply:
If you don't feel strongly about this I would prefer to not do so as we will delete these files (ref.proj, src.proj, tests.proj, dirs.proj, build.proj, package.builds) anyway as we want to define subsets and add the projects as ProjectToBuild arcade extension point items.

Reply:
Sure, don't do anything that's throw away, but understand that dirs.proj is representing key sequencing dependencies, so I suspect these things will need to continue to be grouped somehow.  It'd be nice if they were grouped in a way that permitted folks to perform the common task of "build all inputs needed for building src projects" which is refs and shims.

Reply:
I was already in that file so I did it anyway :)

Yeah that's possible with the subsets. We can follow-up on that later in a few weeks.